cfgov/templates/welcome.html

```diff
@@ -3,14 +3,42 @@
 
 {% block furniture %}
     <div class="content-wrapper">
+
         <h1>{% trans 'Welcome' %} <br/>
-        You've logged in to the consumerfinance.gov editorial tools.	
+        {% if 'build' in request.site.hostname %}
+            You've logged in to the demo Wagtail.    
+        {% else %}
+            You've logged in to the consumerfinance.gov editorial tools.    
+        {% endif %}
         </h1>
-        <h2>Continue on to...</h2>
+
+        <h2>Choose which admin interface you'd like to go to</h2>
         <ul>
-        {% for name, url in destinations %}
-            <li><a href="{{url}}"> {{name}}</a></li>
-        {% endfor %}
+            {% for name, url in destinations %}
+                <li><a href="{{url}}"> {{name}}</a></li>
+            {% endfor %}
         </ul>
+
+        {% if 'build' in request.site.hostname %}
```

Comment:
I kind of agree with @rosskarchner. Why not just expose this everywhere, and then we don't have to worry about special-casing build (especially as we've been exploring options other than build for the Wagtail playground)?